terraform/etl/45-aws-glue-job-active-persons-records.tf

```
@@ -2,16 +2,17 @@ module "active_persons_records_refined" {
   source                         = "../modules/aws-glue-job"
   is_production_environment      = local.is_production_environment
   is_live_environment            = local.is_live_environment
+  count = local.is_live_environment && !local.is_production_environment ? 1 : 0
```

just fixing the alignment
```suggestion
  count                     = local.is_live_environment && !local.is_production_environment ? 1 : 0
```